tests/e2e/integration/auth.spec.js

```diff
@@ -0,0 +1,38 @@
+import path from 'path'
+import { URL } from '../config'
+
+const testKeyPath = path.resolve('test@notary.priv')
+
+describe('Test login page', () => {
+  it('Log in', () => {
+    cy.visit(URL)
+    cy.login(testKeyPath)
+  })
+
+  it('Log out', () => {
+    cy.get('li.el-menu-item:nth-of-type(6)').click({ force: true })
+    cy.contains('Login').should('be.visible')
+  })
+})
+
+describe('Test register page', () => {
+  it('Click sign up button', () => {
+    cy.contains('Sign Up').click()
+    cy.contains('Sign Up').should('be.visible')
+  })
+
+  it('Register new user', () => {
+    cy.get('.el-input__inner').type('jasonstatham')
```

Comment:
![jasonstatham](https://media.giphy.com/media/6LMCkc5oqhdyU/giphy.gif)

Reply:
😂😂😂 
He is cool 😎